guava-collections/src/test/java/org/baeldung/guava/GuavaMultiSetUnitTest.java

```diff
@@ -0,0 +1,67 @@
+package org.baeldung.guava;
+
+import com.google.common.collect.HashMultiset;
+import com.google.common.collect.Multiset;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class GuavaMultiSetUnitTest {
+
+    @Test
+    public void givenMultiSet_whenAddAndRemoveValues_shouldReturnCorrectCount() {
+        Multiset<String> bookStore = HashMultiset.create();
+        bookStore.add("Potter");
+        bookStore.add("Potter");
+        bookStore.add("Potter");
+
+        assertThat(bookStore.contains("Potter")).isTrue();
+        assertThat(bookStore.count("Potter")).isEqualTo(3);
+
+        bookStore.remove("Potter");
+        assertThat(bookStore.contains("Potter")).isTrue();
+        assertThat(bookStore.count("Potter")).isEqualTo(2);
+    }
```

Comment:
This is really two different scenarios. One is that adding makes things present. The other is that removing after adding only decrements the count, rather than removes the object completely.

Reply:
So, I'd recommend splitting into two tests.

Reply:
Done.